AggregateMetrics/AggregateMetrics/AzureWebApp/DefaultCounters.cs

```diff
@@ -53,46 +53,11 @@ public void Initialize()
                 { new FlexiblePerformanceCounterGauge("requestsInApplicationQueue"), new MetricTelemetry() }
             };
 
-            this.rateDefaultCounters = new Dictionary<FlexiblePerformanceCounterGauge, RateCounter>()
+            this.rateDefaultCounters = new Dictionary<RateCounter, MetricTelemetry>()
             {
-                { new FlexiblePerformanceCounterGauge("requestsTotal"), new RateCounter() },
-                { new FlexiblePerformanceCounterGauge("exceptionsThrown"), new RateCounter() }
+                { new RateCounter("requestsTotal"), new MetricTelemetry() },
```

Comment:
You don't need a dictionary - you can have just a list. Counter values would be collected by aggregation module